instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/SqlClientAttributesExtractor.java

```diff
@@ -60,8 +61,11 @@ public void onStart(AttributesBuilder attributes, Context parentContext, REQUEST
     super.onStart(attributes, parentContext, request);
 
     SqlStatementInfo sanitizedStatement = sanitizer.sanitize(getter.rawStatement(request));
+    String operation = sanitizedStatement.getOperation();
     internalSet(attributes, SemanticAttributes.DB_STATEMENT, sanitizedStatement.getFullStatement());
-    internalSet(attributes, SemanticAttributes.DB_OPERATION, sanitizedStatement.getOperation());
-    internalSet(attributes, dbTableAttribute, sanitizedStatement.getTable());
+    internalSet(attributes, SemanticAttributes.DB_OPERATION, operation);
+    if (operation != null && !operation.equals(SQL_CALL)) {
```

Comment:
I think I'd change the condition a bit:
```suggestion
    if (!SQL_CALL.equals(operation)) {
```
In theory, operation can be null and the identifier can be present (although it won't ever happen in practice, I think) -- these two attributes should be independent of each other, let's have the code reflect that.

Reply:
Hey @mateuszrzeszutek, I originally had it this way but it kept complaining about the yoda condition and failing the local build

Reply:
Try rebasing/merging with master; we've disabled the YodaCondition check a couple days ago.

Reply:
@mateuszrzeszutek yep the warning is no longer generated, I made the update